config/routes.rb

```diff
@@ -1,11 +1,12 @@
 Rails.application.routes.draw do
   root 'users#index'
 
-  # Posts routes
-  get '/users/:user_id/posts/:id', to: 'posts#index', as: 'user_post'
-  get '/users/:user_id/posts', to: 'posts#show', as: 'user_posts'
-
-  # Users routes
-  get "/users/:user_id/posts/:id", to: "users#show", as: "user_post_show"
-  get "/users/:user_id/posts", to: "users#index", as: "user_posts_index"
-end
\ No newline at end of file
+  get '/users/:id', to: 'users#show', as: 'user'
+  get '/users', to: 'users#index', as: 'users'
+  post '/users/:user_id/posts/:id/newcomment', to: 'comments#create', as: 'create_new_comment'
+  post '/users/:user_id/posts/:id/newlike', to: 'likes#create', as: 'create_new_like'
+  get '/users/:user_id/posts/new', to: 'posts#new', as: 'user_new_posts'
+  post '/users/:user_id/posts/new', to: 'posts#create', as: 'create_user_posts'
+  get '/users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
+  get '/users/:user_id/posts', to: 'posts#index', as: 'user_posts'
```

Comment:
- [ ] For now, we just needs a pair of routes for creating posts :+1: 
![Screenshot from 2023-10-05 07-13-24](https://github.com/abdozayan12/DataApp/assets/63087491/03ea2d6a-27d8-4bc6-8e5a-f2e4ed13ae2f)